examples/asr/transcribe_speech.py

```diff
@@ -187,15 +196,25 @@ def autocast():
     # transcribe audio
     with autocast():
         with torch.no_grad():
-            transcriptions = asr_model.transcribe(filepaths, batch_size=cfg.batch_size, num_workers=cfg.num_workers)
+            if partial_audio:
+                transcriptions = transcribe_partial_audio(
+                    asr_model=asr_model,
+                    path2manifest=cfg.dataset_manifest,
+                    batch_size=cfg.batch_size,
+                    num_workers=cfg.num_workers,
+                )
+            else:
```

Comment:
Hmm we don't want to disallow users who have offset and duration in manifest from running transcribe (at least whole transcribe can work).  

Do one thing. Check for CTC here then if partial audio send to transcribe partial. If not CTC and still partial audio flag is set - log a warning message saying that RNNT models do not support partial audio transcription (for now) and instead perform the else part.